crates/l2/contracts/src/l1/OnChainProposer.sol

```diff
@@ -390,6 +403,17 @@ contract OnChainProposer is
         emit BatchVerified(lastVerifiedBatch);
     }
 
+    function _checkAndUpdateInclusionQuota(uint16 privileged_transaction_count) private {
+        uint256 pending_count = ICommonBridge(BRIDGE).getPendingTransactionHashes().length;
+        uint16 mimimum_to_include = MIN_INCLUDED_PRIVILEGED_TX > pending_count ? uint16(pending_count) : MIN_INCLUDED_PRIVILEGED_TX;
```

Comment:
This is a bit strange, pending_count is uint256, and we then cast it to uint 16. Why is that?
I imagine having 2**16 - 1 (65535) or more pending transactions is not too far fetched, the conversion will probable lose information about the original value. I'm not saying this is wrong, but I don't see the benefit vs simply setting the type to uint256.